modules/installation-initializing.adoc

```diff
@@ -397,80 +378,7 @@ link:https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-mana
-+
-ifndef::openshift-origin[]
-[source,terminal]
-----
-$ curl -k -u ocpadmin@internal:pw123 \
-https://rhv-env.virtlab.example.com/ovirt-engine/api
-----
-endif::openshift-origin[]
-ifdef::openshift-origin[]
-[source,terminal]
-----
-$ curl -k -u admin@internal:pw123 \
-https://ovirtlab.example.com/ovirt-engine/api
-----
-endif::openshift-origin[]
-+
-... For `Is the oVirt CA trusted locally?`, enter `Yes`, because you have already set up a CA certificate. Otherwise, enter `No`.
-
-... For `oVirt's CA bundle`, if you entered `Yes` for the preceding question, copy the certificate content from `/etc/pki/ca-trust/source/anchors/ca.pem` and paste it here. Then, press `Enter` twice. Otherwise, if you entered `No` for the preceding question, this question does not appear.
-... For `oVirt engine username`, enter the user name and profile of the {rh-virtualization} administrator using this format:
-+
-[source,terminal]
-----
-<username>@<profile> <1>
-----
-<1> For `<username>`, specify the user name of an {rh-virtualization} administrator. For `<profile>`, specify the login profile, which you can get by going to the {rh-virtualization} Administration Portal login page and reviewing the *Profile* dropdown list. Together, the user name and profile should look similar to this example:
-+
-ifndef::openshift-origin[]
-[source,terminal]
-----
-ocpadmin@internal
-----
-endif::openshift-origin[]
-ifdef::openshift-origin[]
-[source,terminal]
-----
-admin@internal
-----
-endif::openshift-origin[]
-+
-... For `oVirt engine password`, enter the {rh-virtualization} admin password.
-... For `oVirt cluster`, select the cluster for installing {product-title}.
-... For `oVirt storage domain`, select the storage domain for installing {product-title}.
-... For `oVirt network`, select a virtual network that has access to the {rh-virtualization} {rh-virtualization-engine-name} REST API.
-... For `Internal API Virtual IP`, enter the static IP address you set aside for the cluster's REST API.
-... For `Ingress virtual IP`, enter the static IP address you reserved for the wildcard apps domain.
-... For `Base Domain`, enter the base domain of the {product-title} cluster. If this cluster is exposed to the outside world, this must be a valid domain recognized by DNS infrastructure. For example, enter: `virtlab.example.com`
-... For `Cluster Name`, enter the name of the cluster. For example, `my-cluster`. Use cluster name from the externally registered/resolvable DNS entries you created for the {product-title} REST API and apps domain names. The installation program also gives this name to the cluster in the {rh-virtualization} environment.
-... For `Pull Secret`, copy the pull secret from the `pull-secret.txt` file you downloaded earlier and paste it here. You can also get a copy of the same {cluster-manager-url-pull}.
-endif::rhv[]
-ifndef::rhv[]
-... Paste the {cluster-manager-url-pull}.
-ifdef::openshift-origin[]
-This field is optional.
-endif::[]
-endif::rhv[]
+endif::[]
```

Comment:
Pretty sure that this line needs to be removed too. There are mismatched ifdefs in this file, and I think removing this line resolves it